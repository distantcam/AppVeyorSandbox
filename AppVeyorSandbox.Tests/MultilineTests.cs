using System.Runtime.CompilerServices;
using ApprovalTests;
using ApprovalTests.Reporters;
using Xunit;

namespace AppVeyorSandbox.Tests
{
    [UseReporter(typeof(DiffReporter))]
    public class MultilineTests
    {
        [MethodImpl(MethodImplOptions.NoInlining)]
        [Fact]
        public void Multiline1()
        {
            Approvals.Verify(@"This
Is
A
Simple
Multiline
String
");
        }
    }
}