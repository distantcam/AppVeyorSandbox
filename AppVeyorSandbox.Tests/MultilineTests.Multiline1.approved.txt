This
Is
A
Simple
Multiline
String
